Device/ARM/ARMCM0/Include/system_ARMCM0.h

```diff
@@ -30,6 +30,13 @@
 extern "C" {
 #endif
 
+/*----------------------------------------------------------------------------
+  Exception / Interrupt Handler Function Prototype
+ *----------------------------------------------------------------------------*/
+typedef void( *pFunc )( void );
+
+extern const pFunc __VECTOR_TABLE[48];
```

Comment:
Unfortunately, moving this declaration to the header file breaks backward compatibility to *all* existing projects using former startup code. We need to check if we can move this back to the implementation. Or we need to revert completely. Sorry. 
cc @GorgonMeducer